### DISCUSSION TITLE
제페토 스크린샷 모드 오류

### DISCUSSION THREAD

**Discussion by LeeYongSu1** [naverz/zepeto-studio-kor#441] [Issue Report (Legacy)]
패키지 버전 : 1.1.5
어플리케이션 버전: 3.10.1
테스트 환경: 안드로이드

안녕하세요

지원해 주신 스크린샷 모드 샘플 프로젝트를 테스트하는 중 캐릭터가 이상하게 움직이는 현상이 있어서 문의 드립니다.

![셀카모드](https://user-images.githubusercontent.com/80555875/155921899-4b164fc6-5879-4a8f-bd11-826bebf813e5.jpg)
![셀카모드2](https://user-images.githubusercontent.com/80555875/155921903-e0519dcd-22af-4faf-83fa-18914ffbf5e0.jpg)

사진에서 보이는 것과 같이 셀카모드에서 셀피 카메라 모드일 때 캐릭터를 움직이면 
애니메이션과 움직임이 이상해지는 현상이 있습니다.

제공해 주신 프로젝트에 가이드 부분 확인 후 따로 건드린 부분이 없고, 다시 다운로드 후 테스트를 진행해도 같은 문제가 발생합니다.

**Comment by hyeonjin-jo** [comment]
안녕하세요.

해당 샘플은 기존 월드에서 사용 가능한 카메라 관련 기능을 제공하는 것에 집중해서 제작이 되었습니다.

셀피모드에서 캐릭터 움직임의 경우 추후 업데이트를 통해서 제공할 예정입니다.

샘플에 대한 관심에 감사드리고 지속적으로 업데이트할 수 있도록 하겠습니다.

감사합니다.

**Reply by LeeYongSu1** [reply]
넵 항상 감사합니다.
